packages/kbn-unified-data-table/src/components/data_table_columns.tsx

```diff
@@ -212,6 +217,7 @@ function buildEuiGridColumn({
         dataView={dataView}
         dataViewField={dataViewField}
         headerRowHeight={headerRowHeight}
+        columnLabel={columnHeaders ? columnDisplayName : undefined}
```

Comment:
I'm not very familiar with this component, but could it be just `columnLabel={columnDisplayName}`?

```jsx
 <DataTableTimeColumnHeaderMemoized
        dataView={dataView}
        dataViewField={dataViewField}
        headerRowHeight={headerRowHeight}
        columnLabel={columnDisplayName}
/>
```

Reply:
i tried that before, but it causes an error